### PR TITLE
register transition

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/ralph/lib/transitions/tests/test_models.py [hunk 1]
```diff
@@ -220,3 +220,14 @@ def test_topological_sort_cycle(self):
         }
         with self.assertRaises(CycleError):
             [a for a in _sort_graph_topologically(graph)]
+
+
+
+class APITest(TransitionTestCase):
+    def test_xxx(self):
+        @transition_action(model=OrderStatus)
+        def example_action(*args, **kwargs):
```
Comment (mkurek) [inline src/ralph/lib/transitions/tests/test_models.py]: this action should take `cls` as a first argument IMO

**Comment by szok** [comment]
:ok:

File: src/ralph/lib/transitions/tests/test_models.py [hunk 2]
```diff
@@ -103,6 +103,31 @@ def test_run_action_during_transition(self):
...
         self.assertTrue(order.go_to_post_office.runned)
 
+    def test_action_is_added_to_model_when_registered_on_model(self):
+        @transition_action(model=OrderStatus)
+        def example_action(cls, *args, **kwargs):
+            pass
+        self.assertTrue(hasattr(OrderStatus, 'example_action'))
+
+    def test_transition_runs_action_registered_on_model(self):
+        @transition_action(model=Order)
```
Comment (mkurek) [inline src/ralph/lib/transitions/tests/test_models.py]: move this function to models.py

File: src/ralph/lib/transitions/decorators.py [hunk 3]
```diff
@@ -22,6 +25,13 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
+
+        if 'model' in kwargs:
+            setattr(kwargs['model'], func.__name__, wrapper)
+            action, _ = Action.objects.get_or_create(name=func.__name__)
```
Comment (mkurek) [inline src/ralph/lib/transitions/decorators.py]: this shouldn't be necessary if transition is registered before post_migrate signal

**Comment by mkurek** [comment]
:ok:

**Pull request merged**